src/panels/config/automation/condition/ha-automation-condition-row.ts

```diff
@@ -6,23 +6,23 @@ import { customElement, property, query, state } from "lit/decorators";
 import { classMap } from "lit/directives/class-map";
 import { fireEvent } from "../../../../common/dom/fire_event";
 import { handleStructError } from "../../../../common/structs/handle-errors";
-import "../../../../components/ha-button-menu";
-import "../../../../components/ha-card";
 import "../../../../components/buttons/ha-progress-button";
 import type { HaProgressButton } from "../../../../components/buttons/ha-progress-button";
-import "../../../../components/ha-icon-button";
+import "../../../../components/ha-button-menu";
```

Comment:
You have merge conflicts because you run a sorting on the imports in each file.

Reply:
Well its on save